src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,13 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        return (bet1 == result1) & (result2 == bet2)
+                ? 2 :
+                (result1 > result2) & (bet1 > bet2)
+                        | (result1 < result2) & (bet1 < bet2)
+                        | (result1 == result2) & (bet1 == bet2)
```

Comment:
why have you decided to use & instead of &&? 
https://stackoverflow.com/a/5564484

Reply:
I use & because I need both inequalities to be true. But I changed operator | to ||.

Reply:
yep, my question was about difference between & and && (bitwise and logical operators)